style.css

```diff
@@ -82,3 +82,209 @@ body {
   margin-top: 11px;
   gap: 17px;
 }
+
+.works-container {
+  display: grid;
+  gap: 10px;
+  padding: 0;
+}
+
+.works-container > div {
+  background-color: white;
+  border-radius: 20px;
+  padding: 20px;
+  margin: 20px;
+}
+
+.works-container ol {
+  display: inline-block;
+  list-style-type: none;
+  padding: 13px;
+}
+
+.works-container li {
+  display: inline-block;
+  padding: 2px;
+}
+
+.works-container h2 {
+  padding-top: 10px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.card-skill-1 {
+  width: 55px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-left: -11px;
+}
+
+.card-skill-2 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  color: #7a869a;
+  margin-left: -19px;
+}
+
+.works-container p {
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tech-skills li {
+  font-size: 11px;
+  line-height: 19px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 5px;
+  padding-left: 12px;
+  padding-right: 12px;
+  margin-top: 15px;
+  margin-bottom: 18px;
+}
+
+button {
+  font-size: 14px;
+  padding: 10px;
+  background: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  color: #396df2;
+}
+
+.About {
+  background: #fff;
+  border-radius: 0 55px 0 0;
+  margin-top: 50px;
+  padding-bottom: 100px;
+}
```

Comment:
Great work @Gleennkar 🤝

- [ ] [Optional] You may try to make your project similar to the Figma design. Please make your font family and font color the same as given in the project requirements
- Your design
![Yours](https://user-images.githubusercontent.com/63917725/177013266-8a1b2372-c1c9-436e-a1a4-206549a16631.PNG)

- Figma design
![Figmas](https://user-images.githubusercontent.com/63917725/177013272-edbbaa6f-e6d0-451e-a3a8-08c976b1c953.PNG)